src/components/template/preview/index.tsx

```diff
@@ -91,18 +101,10 @@ export default function TemplatePreviewPage() {
         </div>
 
         <div className="flex items-center gap-2">
-          <Link href={`/editor/${previewTemplate.templateId}`}>
-            <Button variant="outline" className="w-full rounded-full text-lg">
-              <Edit className="mr-2 size-4" />
-              Edit Template
-            </Button>
-          </Link>
-          <Link href={`/api-playground?template=${previewTemplate.templateId}`}>
-            <Button className="w-full rounded-full text-lg">
-              <Code className="mr-2 size-4" />
-              Try in API Playground
-            </Button>
-          </Link>
+          <Button onClick={() => handleEdit(previewTemplate.templateId as string, previewTemplate.templateType as TemplateType)} variant="outline" className="w-full rounded-full text-lg">
```

Comment:
why is there need to convert the templateId and templateType to their respective types?